cdap-data-fabric/src/main/java/org/apache/tephra/TxConstants.java

```diff
@@ -161,11 +162,11 @@
     /**
      * The maximum time in seconds that a transaction can be used for data writes.
      */
-    public static final String CFG_TX_MAX_LIFETIME = "data.tx.max.lifetime";
+    public static final String CFG_TX_MAX_LIFETIME = Constants.Tephra.CFG_TX_MAX_LIFETIME;
```

Comment:
Tephra classes are copied to cdap-data-fabric. But since we need to access these properties in cdap-tms module which doesn't depend on cdap-data-fabric, I had to copy these properties to CDAP's Constants class and make the Tephra's TxConstants derive from that.